samples/template-operator/README.md

```diff
@@ -34,270 +66,511 @@ Additionally, it hides Kubernetes boilerplate code to develop fast and efficient
+   Assuming the settings from [Prepare & Build module operator image](#prepare--build-module-operator-image) for single-cluster mode, and assuming the following module settings:
+   * hosted at `op-kcp-registry.localhost:8888/unsigned`
+   * generated for channel `stable`
+   * module has version `0.0.1`
+   * module name is `template`
+   * using an insecure (http instead of https for registry communication), local k3d registry on your machine
+   * uses Kyma CLI in `$PATH` under `kyma`
+   * a simple `config.yaml` is present for module configuration with the content
+
+     ```yaml
+     # Samples Config
+     configs:
+     ```
+     _WARNING: Even though this file is empty, it is mandatory for the command to succeed as it will be bundled as layer!
+     kubebuilder projects by default to not have such a file (it is introduced by modularization) and you will need to create one on your own if not already done._ 
+   * the default sample under `config/samples/operator_v1alpha1_sample.yaml` has been adjusted to be a valid CR by setting the default generated `Foo` field instead of a TODO.
+
+     ```yaml
+     apiVersion: operator.kyma-project.io/v1alpha1
+     kind: Sample
+     metadata:
+       name: sample-sample
+     spec:
+       foo: bar
+     ```
+     _WARNING: The settings above reflect your default configuration for a module. If you want to change this you will have to manually adjust it to different configurations. 
+     You can also define multiple files in `config/samples`, however you will need to then specify the correct file during bundling._
+   * The `.gitignore` has been adjusted and following ignores were added
+   
+     ```gitignore
+     # kyma module cache
+     mod
+     # generated dummy charts
+     charts
+     # kyma generated by scripts or local testing
+     kyma.yaml
+     # template generated by kyma create module
+     template.yaml
+     ```
+
+   Now, run
 
    ```sh
-   make module-build
+   kyma alpha create module kyma-project.io/module/sample 0.0.1 . -w --insecure --registry op-kcp-registry.localhost:8888/unsigned
    ```
    
-3. Verify that the module creation succeeded and observe the `mod` folder. It will contain a `component-descriptor.yaml` with a definition of local layers.
+   _WARNING: For external registries (e.g. Google Container/Artifact Registry), never use insecure. Instead specify credentials. More details can be found in the help documentation of the CLI_
    
+   In certain cases (e.g. for demos, testing or in CI), it might make sense to run the `ModuleTemplate` in "control-plane" mode.
+   What this means is that instead of expecting separate control-plane and runtime clusters, we will target a single cluster that holds all infrastructure for the operators and modules.
+
+   To make a setup work that depends on single-cluster mode you will need to adjust the generated `template.yaml` to install the module in the control-plane.
```

Comment:
```suggestion
   To make a setup work in single-cluster mode adjust the generated `template.yaml` to install the module in the Control Plane. This will install all operators and modules in the same cluster.
```